contracts/src/v0.8/functions/dev/1_0_0/ocr/OCR2Abstract.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: MIT
```

Comment:
There are a few minor changes, like not using `TypeAndVersion` and changing how the ownership works.

I'm going to revisit it though. I'm thinking that the Coordinator should still have its own owner right now. It will just be set to the Router Owner to start.

Reply:
Switched to imports

Reply:
Just to clarify, what do you mean by "switched to imports"?
What I meant by my original comment is that OCR2Abstract already exists [here](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/OCR2Abstract.sol) so I am unsure why it is also copied into this current file too (unless I am missing some change?).

Reply:
Bump